index.html

```diff
@@ -2905,6 +2915,152 @@ <h3>Evidence</h3>
+credential</a> to declare that the <a>subject</a> controls one or more
+particular confirmation methods and to bind the <a>claims</a> about the
+<a>subject</a> to one or more of these confirmation methods. In this way, an
+<a>issuer</a> explicitly enables a <a>verifier</a> to validate that the
+<a>holder</a> presenting the <a>verifiable credential</a> has proven control of
+one or more of these confirmation methods when the <a>claims</a> bound to the
+confirmation method are presented.
+        </p>
+
+        <p class="note">
+A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
+credential</a> without validating the confirmation method or to use a different
+mechanism to validate the <a>holder</a> is bound to the presented <a>claims</a>
+if required without impacting liability if not specified by other means such
+as a <code>termsOfUse</code> policy.
+        </p>
+
+        <p>
+This specification defines the <code>confirmationMethod</code> <a>property</a>
+for expressing confirmation method information in a
+<code>credentialSubject</code> in a <a>verifiable credential</a>.
+        </p>
+
+        <p class="note">
+For example, an <a>issuer</a> can include a confirmation method based on public
+key cryptography in the <a>verifiable credential</a>. A <a>holder</a> can
+generate and include a <a>proof</a> with a cryptographic signature in the
+<a>verifiable presentation</a> where the verification key of the cryptographic
+signature is bound to a confirmation method in the embedded <a>verifiable
+credential</a>. A <a>verifier</a> can validate that the <a>holder</a> controls
+the confirmation method by verifying the <a>proof</a> of the <a>verifiable
+presentation</a> using the information in the confirmation method. The
+confirmation method can include the verification key or the type of the
+confirmation method can define that the verification key is inferred by other
+<a>properties</a> in the <a>verifiable credential</a> such as the
+<code>credentialSubject.id</code>.
+        </p>
+
+        <dl>
+          <dt><dfn>confirmationMethod</dfn></dt>
+          <dd>
+If present, the value of the <code>confirmationMethod</code> <a>property</a> is
+one or more confirmation methods each providing enough information for a
+<a>verifier</a> to validate a <a>holder</a> generating a <a>verifiable
+presentation</a> has proven control of a confirmation method bound to <a>claims
+</a> in a <a>verifiable credential</a> in the <a>verifiable presentation</a>.
+Each confirmation method MUST specify its <code>type</code> (for example,
+<code>DIDAuthWithSubjectIdConfirmation2023</code>) and MAY specify an
+<code>id</code>. The precise <a>properties</a> and semantics of each
+confirmation method is determined by the specific
+<code>confirmationMethod</code> type definition.
+          </dd>
+        </dl>
+
+        <p>
+The type <code>DIDAuthWithSubjectIdConfirmation2023</code> defines that a
```

Comment:
I don't believe this or the type below should be defined here. I'd welcome an entry into our not-registry-registry.

Reply:
It's fine to include a link to those specs (and we should!)

Reply:
agree with you, just worried about parity with other extension points